docset/winserver2012r2-ps/hyper-v/Set-VMReplicationServer.md

```diff
@@ -152,7 +152,7 @@ To display a list of certificates in the computer's My store and the thumbprint
 `PS C:\\\> cd cert:\LocalMachine\My`
 `PS C:\\\> dir | format-list`
 
-For more information about certificate stores, see http://technet.microsoft.com//library/cc757138.aspxhttp://technet.microsoft.com//library/cc757138.aspx.
+For more information about certificate stores, see [Certificate stores](https://technet.microsoft.com//library/cc757138.aspx).
```

Comment:
This should be changed to:

https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2003/cc757138(v=ws.10)

Reply:
PR ready when the suggestions have been applied.

cc @velkovb